git/config/install

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+# prototype-kicad-library-ci
+
+# File: git/config/install
+# Description: Git config entries install script for the prototype-kicad-library-ci
+#   repository.
```

Comment:
Please use two tab stops for the indent for consistency with scripts included by other projects (this script's description will be the only one in this project that spans multiple lines until the `ci/scrub-kicad-project-files` script's description is updated).
```suggestion
#       repository.
```